### PR TITLE
Adds trait that reduces the Parquet log level.

### LINKED ISSUE #89: Parquet spams output during testing.
Parquet produces lots of output when running tests such as:
```
Mar 17, 2015 2:54:14 AM INFO: parquet.hadoop.ColumnChunkPageWriteStore: written 33B for [field922] BINARY: 4 values, 2B raw, 2B comp, 1 pages, encodings: [PLAIN_DICTIONARY, BIT_PACKED], dic { 1 entries, 7B raw, 1B comp}
Mar 17, 2015 2:54:14 AM INFO: parquet.hadoop.ColumnChunkPageWriteStore: written 33B for [field923] BINARY: 4 values, 2B raw, 2B comp, 1 pages, encodings: [PLAIN_DICTIONARY, BIT_PACKED], dic { 1 entries, 7B raw, 1B comp}
Mar 17, 2015 2:54:14 AM INFO: parquet.hadoop.ColumnChunkPageWriteStore: written 33B for [field924] BINARY: 4 values, 2B raw, 2B comp, 1 pages, encodings: [PLAIN_DICTIONARY, BIT_PACKED], dic { 1 entries, 7B raw, 1B comp}
```

### PR DESCRIPTION
**Pull request by stephanh**
This closes #89.

### CODE REVIEW

**Comment by stephanh** [comment]
There is still some log output on Travis that I don't seem to be getting locally.

**Comment by stephanh** [comment]
Should be fixed now.

File: core/src/main/scala/au/com/cba/omnia/ebenezer/ParquetLogging.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+/**
+  * Sets the logging level for parquet.
+  *
+  * By default it sets the logging level to SEVERE.
+  * Override `parquetLogLevel` to change the logging level.
+  */
+trait ParquetLogging {
+  /** Override this to change the logging level. */
+  def parquetLogLevel = Level.SEVERE
+
```
Comment (jonjuscorpion666) [inline core/src/main/scala/au/com/cba/omnia/ebenezer/ParquetLogging.scala]: Do we really have any advantage of assigning "Level.Severe "to "parquetLogLevel" .i can see its used only once .cant we set parquetHadoopLogger.setLevel(Level.SEVERE)

Comment (stephanh) [inline core/src/main/scala/au/com/cba/omnia/ebenezer/ParquetLogging.scala]: The idea is that you can override this variable when you mix in the trait to change the log level.

**Comment by vineethvarghese** [comment]
This trait is not really limited to tests. I hope it doesn't find its way into non-test code somewhere :)

:+1:

**Comment by stephanh** [comment]
I was considering having it in non test code. More investigation is needed, but I'm not sure if the parquet log level can be adjusted any other way.

**Comment by vineethvarghese** [comment]
I am skeptical of having this in non test code because it just messes with the normal way we expect to control log output i.e via a configuration. We just need to look back to when @SamRoberts needed logs out of Sqoop for the sqoop zip size issue. 

We pull in many projects with different log libraries, it would be good it we can force them all through slf4j and control all the logging by a slf4j configuration file.

**Comment by stephanh** [comment]
@vineethvarghese there is the jul to slf4j bridge but it says that it has significant performance problems and parquet is quite spammy with its logging.

It's also possible to adjust the logging configuration using `logging.properties` file but it doesn't just load the one on the classpath instead you need to set `-Djava.util.logging.config.file=...`.

**Pull request merged**